hyprpm/src/core/PluginManager.cpp

```diff
@@ -847,17 +868,27 @@ ePluginLoadStateReturn CPluginManager::ensurePluginsLoadState() {
             if (std::find_if(loadedPlugins.begin(), loadedPlugins.end(), [&](const auto& other) { return other == p.name; }) != loadedPlugins.end())
                 continue;
 
-            loadUnloadPlugin(HYPRPMPATH + repoForName(p.name) + "/" + p.filename, true);
+            if (!loadUnloadPlugin(HYPRPMPATH + repoForName(p.name) + "/" + p.filename, true))
+                hyprlandVersionMismatch = true;
```

Comment:
this will still make hyprpm print "loaded" when it didn't?

Reply:
do you prefer to have it print nothing instead, or have it print a warning message?

Reply:
i made it print an info message for now so ill  mark this as resolved, if u want it changed just review it again or smth